app/src/main/java/org/literacyapp/LiteracyApplication.java

```diff
@@ -23,6 +25,9 @@ public void onCreate() {
         db = openHelper.getWritableDatabase();
         daoMaster = new DaoMaster(db);
         daoSession = daoMaster.newSession();
+
+        ScreenOnReceiver screenOnReceiver = new ScreenOnReceiver();
```

Comment:
Tried it. Didn't work. See: http://stackoverflow.com/questions/1588061/android-how-to-receive-broadcast-intents-action-screen-on-off